internal/validate/validate.go

```diff
@@ -98,3 +98,13 @@ func Credentials() error {
 	}
 	return nil
 }
+
+func EnsureFlagHasValidValue(value, flag string, validValues []string) error {
```

Comment:
[nit] on naming, we should read exported names along with the package name, `validate. EnsureFlagHasValidValue` it's a bit of a [stutter](https://stackoverflow.com/questions/45885985/ways-to-avoid-stuttering-in-go-package-and-struct-names), I think we can ditch the `Ensure` part